pages/hold/request/[id]/index.tsx

```diff
@@ -85,40 +94,44 @@ export default function HoldRequestPage({
       const response = await fetch(`${BASE_URL}/api/hold/request/${holdId}`, {
         method: "POST",
         body: JSON.stringify({
-          patronId: patronId.value,
-          source: source.value,
-          pickupLocation: pickupLocation.value,
+          patronId: patronId?.value,
```

Comment:
What case would allow for this request to be made with undefined values for these required fields? I think the optional chaining implies that case is expected.

Reply:
This maybe was unclear. Can you remove the optional chaining? I don't see why we would want to send a hold request that didn't have these values defined.

Reply:
@dgcohen